mdpow/equil.py

```diff
@@ -197,6 +197,7 @@ def __init__(self, molecule=None, **kwargs):
             self.solventmodel = forcefields.get_solvent_model(
                 self.solventmodel_identifier,
                 forcefield=forcefield,
+                model=solventmodel
```

Comment:
.Use keyword `solventmodel` to choose OPLS-AA octanol models. If one wants new parameters, should set `forcefield=OPLS-AA`, `solvent=octanol\wetoctanol` and `solventmodel=octanolnew\wetoctanolnew`

Reply:
Why are there backslashes in `octanol\wetoctanol`? I hope that this is actually `octanol/wetoctanol` or did you just mean to say "`octanol` _or_ `wetoctanol`?

Reply:
That's `octanol` or `wetoctanol`